modules/schemas/src/main/resources/contexts/nexus/core/resource/v0.1.0.json

```diff
@@ -1,11 +1,21 @@
 {
```

Comment:
Since everything (included the nexus/core/standard context) that is managed in nexus will have the resource context added, then I would suggest to have the resource context as a self contained root context with the following content:

```json
{
  "@context": {
    "type": "@type",
    "id": "@id",
    "distribution": {
      "@id": "http://schema.org/distribution"
    },
    "links": {
      "@id": "https://bbp-nexus.epfl.ch/vocabs/nexus/core/terms/v0.1.0/links"
    }
  }
}
```